test/functionalTest/harnessFunctions.sh

```diff
@@ -378,26 +374,24 @@ function brokerStart()
 #
 function brokerStop
 {
-  if [ "$1" == "-v" ]
-  then
-    _verbose=1
-    shift
-  fi
-
   role=$1
-  if [ "$role" == "" ]
-  then
-    role=CB
-  fi
- 
-  vMsg "Stopping broker $1"
 
   if [ "$role" == "CB" ]
   then
     pidFile=$CB_PID_FILE
     port=$CB_PORT
-    vMsg pidFile: $pidFile
-    vMsg port: $port
```

Comment:
The verbose messages were actually useful (not very often, but still) ...
Why remove it?